### PR TITLE
Making things better all the time by refactoring and adding drop

### PR DESCRIPTION
**Pull request by stjohnjohnson**
Also supports prefix and other things.

```
  Usage: tables [options] [command]


  Commands:

    create [tables...]  Create tables
    list                List tables
    drop [tables...]    Drop tables

  Options:

    -h, --help             output usage information
    -V, --version          output the version number
    -r, --region <region>  AWS region to set up the tables in
    -p, --prefix <prefix>  DynamoDB prefix to add to the tables
```

### CODE REVIEW

**Comment by nkatzman** [comment]
👍

File: index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     */
+    constructor(options) {
+        this.options = options || {};
+        this.region = this.options.region || 'us-west-2';
+        vogels.AWS.config.update({ region: this.region });
+    }
+
+    /**
+     * Setup all models for creation
+     */
```
Comment (tkyi) [inline index.js:24]: Add comments for params

File: package.json [hunk 2]
```diff
@@ -43,8 +43,9 @@
     "sinon": "^1.17.4"
   },
   "dependencies": {
+    "async": "^2.0.1",
     "commander": "^2.9.0",
-    "screwdriver-data-schema": "^4.0.0",
+    "screwdriver-data-schema": "^6.0.0",
```
Comment (tkyi) [inline package.json]: Should be `^7.0.0` now

**Comment by tkyi** [comment]
👍 
giphy

**Comment by nkatzman** [comment]
👍 again

**Pull request merged**